web-animations-2/Overview.bs

```diff
@@ -1066,13 +1066,36 @@ condition:
 > *   the <a>animation effect</a> has a <a>parent group</a> and the
 >     <a>parent group</a> is <a>current</a>.
 
-Add the following definition to the list used in determining the phase of
+Add the following definitions to the list used in determining the phase of
 an <a>animation effect</a>.
 
 :   <dfn>effective timeline phase</dfn>
 ::  The associated <a>animation</a>'s <a>hold phase</a> if resolved, otherwise
     the <a>animation</a> timeline's [=timeline phase=].
 
+:   <dfn>effective timeline progress</dfn>
+::  Set based on the first matching condition:
+
+    <dl class="switch">
+
+    :   If the associated <a>animation</a>'s [=timeline duration=] is
+        unresolved,
+    ::  &rarr; unresolved
+
+    :   If the associated <a>animation</a>'s startTime is unresolved,
+    ::  &rarr; <a>inherited time</a>
+
+    :   Otherwise,
+    ::  <code><a>inherited time</a> / <a>animation</a>'s
+        playbackRate + <a>animation</a>'s startTime</code>
```

Comment:
Similarly here, for consistency it might be better to use the model terms instead of the interface member names?

Reply:
Done.